src/pages/account/security/sudo.tsx

```diff
@@ -0,0 +1,112 @@
+import { Identity } from "@fewlines/connect-management";
+import {
+  loggingMiddleware,
+  tracingMiddleware,
+  errorMiddleware,
+  recoveryMiddleware,
+  rateLimitingMiddleware,
+} from "@fwl/web/dist/middlewares";
+import { getServerSidePropsWithMiddlewares } from "@fwl/web/dist/next";
+import type { GetServerSideProps } from "next";
+import React from "react";
+
+import { Container } from "@src/components/containers/container";
+import { Layout } from "@src/components/page-layout";
+import { TwoFA } from "@src/components/pages/two-fa/two-fa";
+import { logger } from "@src/logger";
+import { authMiddleware } from "@src/middlewares/auth-middleware";
+import { sentryMiddleware } from "@src/middlewares/sentry-middleware";
+import getTracer from "@src/tracer";
+import { ERRORS_DATA, webErrorFactory } from "@src/web-errors";
+
+const SudoPage: React.FC<{ primaryIdentities: Identity[] }> = ({
+  primaryIdentities,
+}) => {
+  return (
+    <Layout title="Security">
+      <Container>
+        <TwoFA primaryIdentities={primaryIdentities} />
+      </Container>
+    </Layout>
+  );
+};
+
+const getServerSideProps: GetServerSideProps = async (context) => {
+  return getServerSidePropsWithMiddlewares<{ type: string }>(
+    context,
+    [
+      tracingMiddleware(getTracer()),
+      rateLimitingMiddleware(getTracer(), logger, {
+        windowMs: 300000,
+        requestsUntilBlock: 200,
+      }),
+      recoveryMiddleware(getTracer()),
+      sentryMiddleware(getTracer()),
+      errorMiddleware(getTracer()),
+      loggingMiddleware(getTracer(), logger),
+      authMiddleware(getTracer()),
+    ],
+    "/account/security/sudo",
+    async () => {
+      const webErrors = {
+        identityNotFound: ERRORS_DATA.IDENTITY_NOT_FOUND,
+        connectUnreachable: ERRORS_DATA.CONNECT_UNREACHABLE,
+        notFound: ERRORS_DATA.NOT_FOUND,
+      };
+
+      throw webErrorFactory(webErrors.notFound);
+
+      // const userCookie = await getServerSideCookies<UserCookie>(request, {
+      //   cookieName: "user-cookie",
+      //   isCookieSealed: true,
+      //   cookieSalt: config.cookieSalt,
+      // });
+
+      // if (userCookie) {
+      //   const primaryIdentities = await getIdentities(
+      //     config.managementCredentials,
+      //     userCookie.sub,
+      //   )
```

Comment:
I know this is still WIP, but would be cool to have
```ts
if(!userCookie) {
  //
}
```

Reply:
Noted 👍

Reply:
On this topic, everything is broken atm regarding this. I'm working on a fix ;).